knudev-liquibase/src/main/resources/db/changelog/team_management/2024_10_11_init_db.yaml

```diff
@@ -153,11 +153,6 @@ databaseChangeLog:
                   type: varchar(75)
                   constraints:
                     nullable: false
-              - column:
```

Comment:
I want to maximally reduce now the size of liquibase script. Because we work as 2 people, and easy for us to just re-create all schemas. In future, ofc, we wont, but now we just reduce the size of code